starter_code/queries.md

```diff
@@ -51,33 +57,37 @@
 <!-- Your Code Goes Here -->
 
 ### 12. All the companies founded before 2000 that have an acquisition amount of more than 10.000.000
+filter: {"acquisition.price_amount":{$gte:10000000},founded_year:{$lt:2000}}
 
-<!-- Your Code Goes Here -->
 
 ### 13. All the companies that have been acquired after 2010, order by the acquisition amount, and retrieve only their `name` and `acquisition` field.
-
-<!-- Your Code Goes Here -->
+filter {"acquisition.acquired_year":{$lt:2010}}
+project {name:1,acquisition:1}
+sort {"acquisition.price_amount":1}
 
 ### 14. Order the companies by their `founded year`, retrieving only their `name` and `founded year`.
 
-<!-- Your Code Goes Here -->
+project: {name:1,founded_year:1}
+sort: {founded_year:1}
 
 ### 15. All the companies that have been founded on the first seven days of the month, including the seventh. Sort them by their `acquisition price` in a descending order. Limit the search to 10 documents.
 
-<!-- Your Code Goes Here -->
+filter: {$and: [{founded_day:{$lte:7}}, {founded_month: 1}]}
```

Comment:
filter: {founded_day:{$lte:7}} as it's the first 7 day of what ever month
And search limit to 10